scripts/compile_version_upload_s3.sh

```diff
@@ -81,17 +45,18 @@
 #
 # EXAMPLES
 #
-#  ./scripts/compile_version_upload.sh python 3.9.7
-#  ./scripts/compile_version_upload.sh nodejs 14.17.6
+#  ./scripts/compile_version_upload.sh python-3.9.7
+#  ./scripts/compile_version_upload.sh nodejs-14.17.6
```

Comment:
Note to myself, this may not be required and we can do the same hack as in DOCKER_IMAGE :)